recipes/sleef/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.32.0"
+
+
+class SleefConan(ConanFile):
+    name = "sleef"
+    description = "SLEEF is a library that implements vectorized versions " \
+                  "of C standard math functions."
+    license = "BSL-1.0"
+    topics = ("conan", "sleef", "vectorization", "simd")
+    homepage = "https://sleef.org"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    short_paths = True
+
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def validate(self):
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("shared sleef not supported on Windows, it produces runtime errors")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_STATIC_TEST_BINS"] = False
+        self._cmake.definitions["ENABLE_LTO"] = False
+        self._cmake.definitions["BUILD_LIBM"] = True
+        self._cmake.definitions["BUILD_DFT"] = False
```

Comment:
I regret checking what this option requires

Reply:
what does it mean?

Reply:
DFT is an extra library, by default it's enabled. It means discrete Fourier transform:

https://github.com/shibatch/sleef/blob/c98d92603533bae913ea35ffeab57d0441169290/src/dft/CMakeLists.txt#L332

https://sleef.org/dft.xhtml

Reply:
It's enabled by default in upstream build files, that's true, but I had several build issues with this one. It's not required for libtorch, but someone can enable it later if needed.